.travis.yml

```diff
@@ -5,7 +5,7 @@ env:
  - DJANGO_VERSION=1.9
  - DJANGO_VERSION=1.10
  - DJANGO_VERSION=1.11
- - DJANGO_VERSION=2.0b1
```

Comment:
i think this needs to change back, too. the only thing that should've changed was the classifier, afaict

Reply:
It's fine renaming the test environments, it means less work in the future

Reply:
That's fair, but the version we're installing still needs to be `2.0b1` for now, I think

Reply:
`Django>=2.0b1,<2.1` in tox.ini is doing that, no?

Reply:
oh no I see the logic is duplicated in travis file

Reply:
Yeah, Travis isn't using tox directly, so the error it's returning is
```
pip install -q "Django>=${DJANGO_VERSION},<${DJANGO_VERSION}.99" flake8
  Could not find a version that satisfies the requirement Django<2.0.99,>=2.0
```

Reply:
I changed the version back. Now all build jobs on Travis are green.